indico/modules/categories/forms.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from wtforms.fields import BooleanField, StringField
+
+from indico.util.i18n import _
+from indico.web.forms.base import IndicoForm
+from indico.web.forms.fields import AccessControlListField, PrincipalListField, IndicoProtectionField, EmailListField
+from indico.web.forms.validators import HiddenUnless
+from indico.web.forms.widgets import SwitchWidget
+
+
+class CategoryProtectionForm(IndicoForm):
+    _event_creation_fields = ('event_creation_restricted', 'event_creators', 'event_creation_notification_emails')
+
+    protection_mode = IndicoProtectionField(_('Protection mode'), protected_object=lambda form: form.protected_object)
+    acl = AccessControlListField(_('Access control list'), groups=True, allow_emails=False, allow_external=True,
+                                 allow_networks=True)
+    managers = PrincipalListField(_('Managers'), groups=True)
+    no_access_contact = StringField(_('No access contact'), description=_('Contact information shown when someone '
```

Comment:
we usually put the `description` kwarg in a new line (-> less or no wrapping needed in the string in most cases)